packages/flutter_tools/templates/app/linux.tmpl/CMakeLists.txt.tmpl

```diff
@@ -82,3 +82,14 @@ INSTALL(CODE "
   " COMPONENT Runtime)
 INSTALL(DIRECTORY "${PROJECT_BUILD_DIR}/${FLUTTER_ASSET_DIR_NAME}"
   DESTINATION "${INSTALL_BUNDLE_DATA_DIR}" COMPONENT Runtime)
+
+# Fully re-copy the lib directory on each build to avoid having stale files
+# from a previous install.
```

Comment:
There's only one file, isn't there? I'm not sure how there could be stale files in this structure.